### DISCUSSION TITLE
How to change column formats when connecting to csv files

### DISCUSSION THREAD

**Discussion by momosydney** [dbeaver/dbeaver#8773] [Q&A] [answered]
Hi there,

I’ve connected some csv files to DBeaver and all the columns are set to string formats (abc). I’ve been trying to figure out how to change some columns format to datetime or numeric formats for example but I couldn’t find. I’ve tried to find online tutorial and I couldn’t find.
Thanks for your help in advance

**Comment by kseniiaguzeeva** [comment, marked as answer]
You cannot do it during CSV import. But it is possible to set when a table is exported from a database to another database table (Step **Tables mapping**)
![dbeaver_XPub6WpblM](https://user-images.githubusercontent.com/51405061/82670090-2f644b80-9c45-11ea-936a-e65d5bc9d3ed.png)
Press **Columns** btn for it.

Please reopen the ticket if you have additional questions.
